e2e/rbd.go

```diff
@@ -1240,6 +1240,31 @@ var _ = Describe("RBD", func() {
 				validateRBDImageCount(f, 0)
 			})
 
+			By("create a PVC and Bind it to an app for mapped rbd image with options", func() {
+				err := deleteResource(rbdExamplePath + "storageclass.yaml")
+				if err != nil {
+					e2elog.Failf("failed to delete storageclass with error %v", err)
+				}
+				err = createRBDStorageClass(f.ClientSet, f, nil, map[string]string{
```

Comment:
In this case you need to use krbd mounter (set, or check that it is configured). Maybe add a test for using rbd-ndb too?

Reply:
there are no tests for nbd driver as of now, when we start supporting nbd we can add/modify tests

Reply:
In that case, maybe just force kRBD to be used in this test. Once rbd-nbd gets added, this test should not suddenly start to fail?

Reply:
by default cephcsi uses rbd, once we start supporting nbd in CI we need to make code changes in E2E to pass krbd and nbd mounter I think this should be fine.